docs/docs/development/setup/docker-vscode.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Docker+VS Code Setup
+sidebar_label: Docker
+---
+
+:::danger
+The Docker environment will **NOT** run on arm CPUs like the Raspberry Pi. You must use the native environment if using an arm CPU.
+:::
+
+:::note Windows Users
+Please note the zmk builds can run slower with Docker on Windows if you don't use the WSL2 filesystem to store files. Build times can take 3-5 minutes on slower hardware without using the WSL2 filesystem. If you run into performance problems you can checkout the zmk sources inside a WSL2 environment and use `code .` inside the WSL2 environment to open the sources. This can make builds run as fast as 20s.
```

Comment:
If the path of least resistance is installing Docker for Windows and letting that work the WSL2 magic behind the scenes, maybe we ought to recommend the happiest path upfront. Can also link to a section at the end that talks about the finer points of WSL and IO perf.

Reply:
I think we can leave this note but move it after the setup steps.

 @innovaker was the one driving this add initially and I don't think it detracts from the rest of the information on the page.